### PR TITLE
Add hoarder extension

### PR DESCRIPTION
**Pull request by kdurek**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/hoarder/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Hoarder
+
+Manage hoarder bookmarks
```
Comment (pernielsentikaer) [inline extensions/hoarder/README.md:3]: Could you describe a bit more about this extension

File: extensions/hoarder/package.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    }
+  ],
+  "preferences": [
+    {
+      "name": "apiUrl",
+      "type": "textfield",
+      "title": "API URL",
+      "description": "The URL of the Hoarder API",
+      "required": true,
+      "placeholder": "https://hoarder.app"
```
Comment (pernielsentikaer) [inline extensions/hoarder/package.json:24]: Can you host it yourself?

File: extensions/hoarder/src/types.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+export type Preferences = {
+  apiUrl: string;
+  apiKey: string;
+};
```
Comment (pernielsentikaer) [inline extensions/hoarder/src/types.ts:4]: This can be removed since it's already a part of raycast-env.d.ts

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you 🙂

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/hoarder/package.json (hunk 2 above)
Comment (kdurek) [reply extensions/hoarder/package.json:24]: What do you mean, is it self hostable? I provided placeholder just for the sake it should be url :D

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Could you add at least one [screenshot](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots) of the extension 🙂

File: extensions/hoarder/package.json (hunk 2 above)
Comment (kdurek) [reply extensions/hoarder/package.json:24]: Oh okay! I mean it's only self-hostable for now at least, I just missed ability to find my bookmarks in raycast, prob gonna extend this extension forward

**Comment by kdurek** [comment]
> Could you add at least one [screenshot](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots) of the extension 🙂

Yes, gonna prepare something!

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**